api/src/test/java/com/datastrato/gravitino/TestTableChange.java

```diff
@@ -96,7 +96,8 @@ public void testAddColumnWithPosition() {
   public void testAddColumnWithNullCommentAndPosition() {
     String[] fieldName = {"Middle Name"};
     Type dataType = Types.StringType.get();
-    AddColumn addColumn = (AddColumn) TableChange.addColumn(fieldName, dataType, null, null);
+    AddColumn addColumn =
+        (AddColumn) TableChange.addColumn(fieldName, dataType, null, (ColumnPosition) null);
```

Comment:
Thanks for catching this! Added to most of the `addColumn` operations. I didn't add for `testAddColumnNotEqualsAndHashCode` and `testAddColumnEqualsAndHashCode` since their main goal is to test for equality and hash code.